sympy/geometry/parabola.py

```diff
@@ -286,36 +285,35 @@ def intersection(self, o):
         Examples
         ========
 
-        >>>from sympy import Parabola, Point, Ellipse, Line, Segment2D
-        >>>p1 = Point(0,0)
-        >>>l1 = Line(Point(1, -2), Point(-1,-2))
-        >>>
-        >>>parabola1 = Parabola(p1, l1)
-        >>>parabola1.intersection(Ellipse(Point(0, 0), 2, 5))
+        >>> from sympy import Parabola, Point2D, Ellipse, Line, Segment2D
```

Comment:
It's not necessary to use the explicit '2D' in the names. None of the other doctests (except for those in point.py and a couple stragglers in util.py) do so.